markdown/manual.md

```diff
@@ -1,220 +1,245 @@
 
 Potential professional development opportunities include, but are not limited to:
 * Attending relevant professional conferences.
 * Tuition for relevant classes.
 * Books on relevant topics.
 
-To seek approval for a professional development opportunity, please submit the following items to your supervisor, who will review and discuss them with you, and share them with the Board of Directors.
-* A short memo outlining the professional development opportunity you are seeking, how it would support your growth, how it would benefit the organization.
+To seek approval for a professional development opportunity, please submit the following items to your supervisor, who will review and discuss them with you, and if requiring an expenditure greater than $999, share them with the Board of Directors for approval.
+* A description of the professional development opportunity, how it would support your growth, and how it would benefit the organization.
 * The total cost of the opportunity, and any relevant payment dates and methods.
 * The time the opportunity would require (dates, times, including travel) as well as the location.
 
-
 ## Schedule, Hours & Vacation
 
-### Office Hours, Daily Schedule & Absences
+### Our Schedule
 
-Employees are expected to work a minimum of 40 hours per week, but we are flexible on where and when that work gets done.
+Employees are expected to work 40 hours per week, but we are flexible on where and when that work gets done.
 
-Employees should check in on Teamwork by 11 AM Eastern Time each week day with an update of their work plan for the day. Communication is vital to the success of our team; therefore, team members are expected to be reasonably accessible by phone, Slack, and email between the hours of 11 AM–4 PM.
+Employees are expected to attend all meetings to which they have been invited as a "required" participant. Communication is vital to the success of our team; therefore, team members are expected to be reasonably accessible by phone, Slack, and email between the hours of 11 AM–4 PM.
 
-Employees are able to work from home and/or remotely, as long as they are able to participate in all necessary meetings.
+Employees are able to work from home and/or remotely and the organization will make all efforts to ensure a comparable work experience regardless of where an employee is located.
 
-Any time that you won't be available during usual office hours (9 PM–5 PM)—for appointments, sick time, vacation, etc.—should be put on the calendar, and an email must be sent to the employee's supervisor.  For non-emergency medical absences, please notify your supervisor at least 48 hours in advance.
-
-We track time using Harvest. Hours worked must be recorded in Harvest by the end of each week (Sunday evening).
+Any time that you won't be available during the regular work day (9 PM to 5 PM locally) should be put on your calendar the organization's calendar.
```

Comment:
> should be put on your calendar the organization's calendar.

Reply:
those pesky conjunctions!

Reply:
Also, 9 **PM** to 5 PM should probably be 9 **AM** to 5 PM.